bot/exts/filters/antispam.py

```diff
@@ -297,10 +297,13 @@ async def on_message_edit(self, before: Message, after: Message) -> None:
         self.cache.update(after)
 
 
-def validate_config(rules_: Mapping = AntiSpamConfig.rules) -> Dict[str, str]:
+def validate_config(rules_: Mapping = None) -> Dict[str, str]:
```

Comment:
Is it not possible to move the `AntiSpamConfig.rules.dict()` as the default value, similar to what was there before?

Reply:
That's what I did in the first place, but flake8 didn't like that.
Here's what it says

```error
B008 Do not perform function calls in argument defaults.  The call is performed only once at function definition time. All calls to your function will reuse the result of that definition-time fun
ction call.  If this is intended, assign the function call to a module-level variable and use that variable as a default value.
```
Which is why I did that, I can either leave it the way it is or tell flake8 to ignore it, WDYT ?

Reply:
Well, I ended up making it a module level var here db665dcd1ccdd89ff4bd1c20bfb3c340418c86c3